test/contrib/test_control_flow.py

```diff
@@ -78,3 +79,33 @@ def body_fn(z, val):
     with trace(), seed(rng_seed=0), substitute(data={"z": data}):
         zs = model()
     assert_allclose(zs, data)
+
```

Comment:
We might need to run the time series notebook again to see if the results look reasonable (if anything, it should be better)?

Reply:
That tutorial does not have time-dependent latent variables so it should be the same IMO.